README.md

```diff
@@ -496,6 +496,8 @@ docker compose build
 docker compose run client
 ```
 
+> Note that if you get an error like "'name' does not match any of the regexes: '^x-'" then you will need to upgrade your version of docker-compose.
```

Comment:
i think the hyphenated use is now deprecated / old.

```suggestion
> Note that if you get an error like "'name' does not match any of the regexes: '^x-'" then you will need to upgrade your version of docker compose.
```

Reply:
Done